tendrl/gluster_integration/objects/native_events/__init__.py

```diff
@@ -0,0 +1,33 @@
+from tendrl.commons import objects
+
+
+class NativeEvents(objects.BaseObject):
+    def __init__(
+        self,
+        context=None,
+        processed=None,
+        recovery_processed=None,
+        message=None,
+        severity=None,
+        ttl=None,
+        current_value=None,
+        *args,
+        **kwargs
+    ):
+        super(NativeEvents, self).__init__(*args, **kwargs)
+
+        self.context = context
+        self.processed = processed
+        self.recovery_processed = recovery_processed
+        self.message = message
+        self.severity = severity
+        self.ttl = ttl
+        self.current_value = current_value
+        self.value = 'clusters/{0}/native_events/{1}'
+
+    def render(self):
+        self.value = self.value.format(
+            NS.tendrl_context.integration_id,
+            self.context
```

Comment:
strip the context of any spaces and forward backward slashes

Reply:
ack